exercises/Exercise4-MergingAndReReRe.md

```diff
@@ -14,7 +14,7 @@ Switched to branch 'exercise4'
 
 ### Exercise
 1. Merge the `exercise3` branch into `exercise4`, and look at the git log.
-2. Use `git reset --hard` to reset your `exercise4` branch back one commit, then use the `--no-ff` option to `git merge` to merge `exercise3` again. Look at the git log, how is it different from the last step?
+2. Use `git reset --hard HEAD^` to reset your `exercise4` branch back one commit, then use the `--no-ff` option to `git merge` to merge `exercise3` again. Look at the git log, how is it different from the last step?
 3. Make two conflicting changes to `hello.txt` in two different branches.
 4. Enable ReReRe, then merge one branch into the other.
 5. Backup again with `git reset --hard`, then attempt the merge again. Notice how ReReRe automatically resolves the conflict for you.
```

Comment:
I think we can add it there too:

```suggestion
5. Backup again with `git reset --hard HEAD^`, then attempt the merge again. Notice how ReReRe automatically resolves the conflict for you.
```

Reply:
I think so too because in both cases the intent is to go back one commit, thanks!